SkyBlock/SKYBLOCK.SK/Events.sk

```diff
@@ -248,23 +261,6 @@ on spawn of a phantom:
   if {SB::config::phantomspawn} is true:
     cancel event
 
-#
```

Comment:
This part is no longer needed, now, the teleport itself is cancelled, if the nether portal is disabled and the player wants to use it.